.travis.yml

```diff
@@ -51,12 +51,12 @@ matrix:
     os: linux
     group: deprecated
     before_script:
-    - export CC="gcc-4.9"
```

Comment:
Is this expected to be in here?

Reply:
We aren't using clang-4.0 in any other build matrix, any objections to using that here (we use gcc 4.9 in other builds)?

Reply:
We use clang-4.0 in the integration tests, but as we use gcc-4.9 in the manylinux1 builds I do not care much about the choice of compiler here as long as we test Python+old gcc together in some build.

Reply:
:+1: